starter_code/views/page-movie.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="page">
+    <div class="movie_image">
+        <img src="{{movie.image}}" alt="">
+    </div>
+    <div class="movie_text">
+        <a href="/movies">Go back</a>
+        <h2>{{movie.title}}</h2>
+        <h3>Director: {{movie.director}}</h3>
+        <h4>Stars</h4>
+        <ul class="movie_stars">
+            {{movie.stars}}
+        </ul>
+        <p>{{movie.description}}</p>
+        <p>{{movie.showtimes}}</p>
```

Comment:
Same as the previous comment, the movie.showtimes key is an array, you need to loop over it to display each elements of the array.
Displaying an array works, but each element would be seperated with a comma.